src/test/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentServiceSpec.scala

```diff
@@ -207,4 +207,136 @@ class ConsignmentServiceSpec extends AnyFlatSpec with MockitoSugar with ResetMoc
     body.name shouldBe mockBody.head.name
   }
 
+  "getConsignments" should "return all the consignments from the cursor to the limit" in {
+    val consignmentId1 = UUID.fromString("20fe77a7-51b3-434c-b5f6-a14e814a2e05")
+    val consignmentId2 = UUID.fromString("fa19cd46-216f-497a-8c1d-6caaf3f421bc")
+    val consignmentId3 = UUID.fromString("614d0cba-380f-4b09-a6e4-542413dd7f4a")
+
+    val consignmentRowParams = List(
+      (consignmentId1, "consignment-ref1", 1L),
+      (consignmentId2, "consignment-ref2", 2L),
+      (consignmentId3, "consignment-ref3", 3L)
+    )
+
+    //Returned rows from repository are 'limit + 1' so additional row included to represent correct response
+    val consignmentRows: List[ConsignmentRow] = consignmentRowParams.map(p => createConsignmentRow(p._1, p._2, p._3))
+
+    val limit = 2
+
+    val mockResponse: Future[Seq[ConsignmentRow]] = Future.successful(consignmentRows)
+    when(consignmentRepoMock.getConsignments(limit + 1, "consignment-ref1")).thenReturn(mockResponse)
+
+    val response: PaginatedConsignments = consignmentService.getConsignments(limit, "consignment-ref1").futureValue
+
+    verify(consignmentRepoMock, times(1)).getConsignments(limit + 1, "consignment-ref1")
```

Comment:
Minor thing, but you don't strictly need to call `verify` when your test is checking something without side-effects (in this case, fetching data from the repository). It's a repeat of the setup on line 227, and the interesting assertions are the ones below - does it return the right data?